src/dbnode/server/server.go

```diff
@@ -1165,6 +1167,81 @@ func kvWatchEncodersPerBlockLimit(
 	}()
 }
 
+func kvWatchQueryLimit(
+	store kv.Store,
+	logger *zap.Logger,
+	limits limits.QueryLimits,
+) {
+	value, err := store.Get(kvconfig.QueryLimits)
+	if err == nil {
+		protoValue := &kvpb.QueryLimits{}
+		err = value.Unmarshal(protoValue)
+		if err == nil && protoValue != nil {
+			updateQueryLimits(logger, limits, protoValue)
+		}
+	} else if !errors.Is(err, kv.ErrNotFound) {
```

Comment:
Think you're dropping the error here if it's not ErrNotFound

Reply:
Well I think in the general case we don't have this set since it is more for overriding. So the behavior we want is to just skip over if the key isn't present.

Reply:
I can log info though

Reply:
Actually I think we should just remain consistent w/ all of these other k/v settings. We Warn on actual error and ignore and go w/ config value otherwise.